src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -949,21 +949,21 @@ private bool CheckPropertyValueKind(SyntaxNode node, BoundExpression expr, BindV
             ImmutableArray<BoundExpression> args,
             ImmutableArray<RefKind> argRefKinds,
             ImmutableArray<int> argsToParamsOpt,
-            uint scopeOfTheContainingExpression)
+            uint scopeOfTheContainingExpression,
+            bool isRefEscape
+        )
         {
             // SPEC: (also applies to the CheckInvocationEscape counterpart)
             //
             //            An lvalue resulting from a ref-returning method invocation e1.M(e2, ...) is ref-safe - to - escape the smallest of the following scopes:
```

Comment:
>smallest [](start = 129, length = 8)

Nit: "smallest" is ambiguous. Consider using "narrowest" here and a couple of lines below.

Reply:
That is the terminology from the spec proposal. I think we should fix it there. 
Here it is less important, since it is just a comment. Will fix when the spec is updated. (hopefully there will be no changes to rules too :-)